api/roleAccess/dtos/to-role-assignment-dto.ts

```diff
@@ -1,4 +1,5 @@
 /* tslint:disable:object-literal-sort-keys */
+import { UserInfo } from 'auth/interfaces/UserInfo';
```

Comment:
need relative folder '../auth' for example

Reply:
Changes pushed